src/components/Form/Form.md

```diff
@@ -0,0 +1,66 @@
+```jsx
+import { TextInput, Spacer, Button, InfoMessage } from '../index'
+import validateMinMaxFactory from '../../validators/validateMinMax'
+import validateTruthy from '../../validators/validateTruthy'
+;<Form
+  config={{
+    formName: 'Styleguidist example form',
+    formId: '1',
+    inputs: {
```

Comment:
ok, I can sort of see why you want to move towards `^input^field`. Do we then consider certain, ahem, fields NOT fields e.g. I suppose here a `<Button />` is not considered a "field" per se. I suppose we'd want to be crystal clear what a field is so we can make it easy for team to understand. I suppose you could say "If you have something like a button that has no `validator` or `TBD???`, we consider that a component and not really a field." Let's contemplate how to make this clear.

Reply:
I think the folder structure is enough. I moved button select group to the "inputs" folder, with your blessing i'd like to just change that folder to be called "fields" and that should be enough. Tell me if you want this change to get made. 

`<Button.Medium.Black type="submit">Submit</Button.Medium.Black>` feels very nice to me. it's a form so submit should work right?